ocaml/typing/typecore.ml

```diff
@@ -8333,7 +8356,6 @@ and type_let ?check ?check_strict ?(force_toplevel = false)
   let spatl = List.map (pat_modes ~force_toplevel rec_mode_var) spatl in
   let attrs_list = List.map (fun (attrs, _, _, _) -> attrs) spatl in
   let is_recursive = (rec_flag = Recursive) in
-
```

Comment:
I assume this deletion is accidental? Even if not, I'd avoid making style changes.